setup.py

```diff
@@ -88,7 +88,7 @@
     'jsonfield==1.0.3',
     'Markdown==2.6.2',
     'pytz==2015.4',
-    'requests==2.7.0',
+    'requests==2.8.1',
```

Comment:
Travis already has Requests==2.8.1 installed, so installing Shoop was downgrading it without any good reason.